packages/logger/index.js

```diff
@@ -0,0 +1,123 @@
+const logger = require('../dd-trace/src/log')
+// path to require tracer logger
+
+const https = require('https')
+const coalesce = require('koalas')
+
+class V2LogWriter {
+  // Note: these attribute names match the corresponding entry in the JSON payload.
+  constructor ({ ddsource, hostname, service, site, apiKey, interval, timeout }) {
+    this.ddsource = ddsource
+    this.hostname = hostname
+    this.service = service
+    this.interval = interval
+    this.timeout = timeout
+    this.buffer = []
+    this.buffer_limit = 1000
+    this.apiKey = apiKey
+    this.endpoint = '/api/v2/logs'
+    this.site = coalesce(site, 'datadoghq.com')
+    this.intake = `http-intake.logs.${this.site}`
+    this.headers = {
+      'DD-API-KEY': this.apiKey,
+      'Content-Type': 'application/json'
+    }
+    this.timer = setInterval(() => {
+      this.flush()
+    }, this.interval)
+
+    logger.debug(`"started log writer to ${this.url}"`)
```

Comment:
I think this would log the contents of the function instead of calling it. To call it there should be parens at the end: `${this.url()}`